combustion-android-ble/src/main/java/inc/combustion/framework/ble/uart/meatnet/NodeReadFirmwareRevisionResponse.kt

```diff
@@ -0,0 +1,77 @@
+/*
+ * Project: Combustion Inc. Android Framework
+ * File: NodeReadFirmwareRevisionResponse.kt
+ * Author: https://github.com/angrygorilla
+ *
+ * MIT License
+ *
+ * Copyright (c) 2022. Combustion Inc.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+package inc.combustion.framework.ble.uart.meatnet
+
+import inc.combustion.framework.ble.getLittleEndianUInt32At
+
+internal class NodeReadFirmwareRevisionResponse (
+    val serialNumber: UInt,
```

Comment:
Same, this number converted to a string -- and formated.  There is likely another example in of this when we handle advertising data.

Reply:
updated